test/unit/components/email-step.test.jsx

```diff
@@ -21,12 +22,17 @@ jest.mock('../../../src/lib/validate.js', () => (
 const EmailStep = require('../../../src/components/join-flow/email-step.jsx');
 
 describe('EmailStep test', () => {
+    const defaultProps = () => ({
```

Comment:
I'm not clear on why you need this. You are passing a different sendAnalytics function when you actually test it, right?

Reply:
Nope. In the test that actually tests it, I'm just testing that the prop function gets called with the right parameters.  The defaultProps is here because I didn't guard the call in componentDidMount with a check for whether the prop exists so I have to make it exist.

Reply:
OK